.travis.yml

```diff
@@ -0,0 +1,95 @@
+language: python
+python:
+  - "3.8.2"
+
+os: linux
+dist: xenial
+
+services:
+  - docker
+
+import:
+  - Alfresco/alfresco-process-tools:.travis.helm.yml@master
+  - Alfresco/alfresco-process-tools:.travis.kubernetes_install.yml@master
+  - Alfresco/alfresco-process-tools:.travis.docker_login.yml@master
+
+env:
+  global:
+    - TRAVIS_WAIT_TIMEOUT=${TRAVIS_WAIT_TIMEOUT:-30}
+    - BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-${TRAVIS_BRANCH}}
+    - HELM_REPO_BASE_URL=https://kubernetes-charts.alfresco.com
+    - HELM_REPO=incubator
+    - KUBERNETES_VERSION=1.18.4
+    - HELM_VERSION=${HELM_VERSION:-3.2.4}
+
+branches:
+  only:
+    - OPSEXP-115
+
+before_install:
+  - |
+    pip install --upgrade awscli
+    curl -o aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.16.8/2020-04-16/bin/linux/amd64/aws-iam-authenticator
+    curl -o aws-iam-authenticator.sha256 https://amazon-eks.s3.us-west-2.amazonaws.com/1.16.8/2020-04-16/bin/linux/amd64/aws-iam-authenticator.sha256
+    openssl sha1 -sha256 aws-iam-authenticator
+    chmod +x ./aws-iam-authenticator
+    mkdir -p $HOME/bin && cp ./aws-iam-authenticator $HOME/bin/aws-iam-authenticator && export PATH=$PATH:$HOME/bin
+    export PATH=$PATH:$HOME/bin
+    aws eks update-kubeconfig --name acs-cluster --region=eu-west-1
+
+before_script:
+  - |
+    REPO_NAME=${TRAVIS_REPO_SLUG##*/}
+    PROJECT_NAME=alfresco-content-services
+    helm repo add activiti https://activiti.github.io/activiti-cloud-helm-charts
```

Comment:
Out of interest why do we need to add the activiti helm repo when we're only deploying ACS?

Reply:
ill remove it, we dont.